source/index.js

```diff
@@ -285,9 +286,14 @@ class CacheableLookup {
 
 		this._tickLocked = true;
 
-		setTimeout(() => {
+		const interval = setTimeout(() => {
```

Comment:
```suggestion
		const timeout = setTimeout(() => {
```

It's not an interval, it's a timeout :)